src/views/Player.vue

```diff
@@ -200,12 +213,18 @@ export default {
 
       // TODO: make this better -> not using the
       // benefits of Vue here
-      var index = Number(ivq.id)
+      var currQuesIndex = Number(ivq.id)
 
       // Checking if the object is empty or not.
       // If empty, push the answer. Otherwise don't.
-      if(Object.keys(this.answers[index]).length === 0)
-        this.answers[index] = answer
+      if(Object.keys(this.answers[currQuesIndex]).length === 0){
+        this.answers[currQuesIndex] = answer
+      }
+
+      this.submitted[currQuesIndex] += 1;
+
+      this.skipped[currQuesIndex] = 0;
+      this.revised[currQuesIndex] = 0;
```

Comment:
Well this is good no? If the user did all three, we should have all three informations available. It might indicate somewthing about the question that users have to revise, then submit and all. Don't set to zero.

Reply:
We won't get to know the order with which they did those actions, but haan we can add that somehow later on if needed. Theek hai I'll remove the 0!

Reply:
Haan. data manipulate nahi karne ka! :P

Reply:
+1 - should not set to 0